src/com/lilithsthrone/game/character/GameCharacter.java

```diff
@@ -26915,12 +26915,21 @@ public String incrementBreastStoredMilk(float increment) {
 	}
 	// Regen:
 	public FluidRegeneration getBreastLactationRegeneration() {
+		if (!Main.game.isLactationContentEnabled()) {
+			return FluidRegeneration.ZERO_SLOW;
+		}
 		return body.getBreast().getLactationRegeneration();
 	}
 	public int getBreastRawLactationRegenerationValue() {
+		if (!Main.game.isLactationContentEnabled()) {
+			return 0;
+		}
```

Comment:
Unsure if the following is acceptable or bad (though the situation may be rather unlikely):
1. Start game or load game with lactation content enabled
2. Set lactation values to an amount
3. Disable lactation content
4. Trigger a call to `incrementBreastLactationRegeneration`
5. Reenable lactation content
6. Lactation values will be strange because (4) incremented form zero, instead of from current value

True for cum storage, and crotch milk regen, and anything else whose `increment*` uses the getter.

Reply:
Will review if it is possible to call `incrementBreastLactationRegeneration` while lactation is disabled. I wouldn't expect it to be but then this is written in spaget

Reply:
'Possible' includes mods, unfortunately, so yes it is possible.

Reply:
These checks were made redundant by changes to the UI so I have removed them